### PR TITLE
Minor tweaks.

### PR DESCRIPTION
**Pull request by dkorolev**

### CODE REVIEW

File: Reflection/test.cc [hunk 1]
```diff
@@ -43,22 +16,99 @@ CURRENT_STRUCT(DerivedFromFoo, Foo) { CURRENT_FIELD(bar, Bar); };
...
+  EXPECT_EQ(7u, Reflector().KnownTypesCountForUnitTest());
+}
+
+TEST(Reflection, TypeID) {
+  using namespace reflection_test;
+  using current::reflection::ReflectedType_Struct;
+
+  // TODO(dkorolev): Migrate to `Polymorphic<>` and avoid `dynamic_cast<>` here.
+  const ReflectedType_Struct& bar = dynamic_cast<const ReflectedType_Struct&>(*Reflector().ReflectType<Bar>());
+  EXPECT_EQ(9010000001031372545, static_cast<uint64_t>(bar.fields[0].first->type_id));
```
Comment (mzhurovich) [inline Reflection/test.cc]: Add `u` literal, pls.

Comment (dkorolev) [inline Reflection/test.cc]: Already `ull`-ed it, work in progress ...

Comment (mzhurovich) [inline Reflection/test.cc]: Ok! Then I'm merging this PR as is.

**Comment by dkorolev** [comment]
Thank you @mzhurovich :+1:

**Pull request merged**